utils/ansible/ftest/vars/Rocky8.yml

```diff
@@ -32,9 +32,11 @@ daos_base_deps:
 daos_dev_deps:
   rpms:
     - daos-raft-devel
+    - daxctl-devel
```

Comment:
nit: unrelated changes should probably be in their own PR to separate concerns

Reply:
I agree this should be separate. And since it's just ansible - not used in CI - we don't need to run all the Jenkins validation for a separate PR that just updates ansible

Reply:
- [ ] Remove the ansible script commit

Reply:
> * [x]  Remove the ansible script commit

Fixed with commit 6b1c1504f